src/libsync/propagatedownload.cpp

```diff
@@ -355,6 +355,8 @@ void PropagateDownloadFile::start()
     if (_item->_type == ItemTypeVirtualFile) {
         auto fn = propagator()->getFilePath(_item->_file);
         qCDebug(lcPropagateDownload) << "creating virtual file" << fn;
+
+        // NOTE: Other places might depend on shape of placeholder files (like csync_update)
```

Comment:
shape of a file is what? rectangle? square? circle?

Reply:
everyone knows it's a rectangle, doh :P

Reply:
yeah  s/shape/content/  ?

Reply:
Yeah, agree :)